nomad/structs/csi.go

```diff
@@ -167,13 +168,14 @@ type CSIVolume struct {
 type CSIVolListStub struct {
 	ID                  string
 	Namespace           string
+	Name                string
+	ExternalID          string
```

Comment:
We have `ID`, `Name`, and `ExternalID`. Can we have a short docstring explaining the difference?